packages/core/content-manager/server/controllers/validation/model-configuration.js

```diff
@@ -22,6 +26,13 @@ module.exports = (schema, opts = {}) =>
 const createSettingsSchema = (schema) => {
   const validAttributes = Object.keys(schema.attributes).filter((key) => isListable(schema, key));
 
+  // TODO V5: Refactor non visible fields to be a part of content-manager schema
+  const model = strapi.getModel(schema.uid);
+  const nonVisibleWritableAttributes = intersection(
```

Comment:
Just noticed localizations end up in that list 🤔 I'm not sure that makes sense but not sure we can set them as non writable. Do you mind checking that ?

Reply:
👍 I will check it. Do you think it's a problem if we allow to sort by localizations & locale?

Reply:
Sorting by those fields is already allowed as of today.

Reply:
Hum 🤔 by locale yes localizations I'm surprised but if that's the case then there is no reason you can do it in the API and not the Admin even if that would make much sense to filter by localizations anyway